src/Lykke.Job.Messages/QueueConsumers/SmsQueueConsumer.cs

```diff
@@ -82,10 +84,14 @@ private void InitQueues()
                 nameof(HandleSmsRequestAsync),
                 $"SMS: Phone confirmation. Receiver: {request.PhoneNumber}, UTC: {DateTime.UtcNow}");

```

Comment:
Should we hide personal data or at least obscure it?

Reply:
I don't thank that this is a subject of current pull request, but we can escalate this question and fix it in future releases. Anyway phone number is logged now so we don't reveal any additional information by current update